Karma.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.31402.337
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Karma", "Karma.csproj", "{FCFA426F-8BB0-445E-A3B8-3489370241A3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Karma", "Karma.csproj", "{FCFA426F-8BB0-445E-A3B8-3489370241A3}"
```

Comment:
what is this change?

Reply:
It changed automatically after I created a file(I don't remember which one)